src/components/App.js

```diff
@@ -0,0 +1,21 @@
+import '../App.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div>
+      <div className="App">
+        Hello World!, This is a Heroku Test
+      </div>
+      <Display />
+      <ButtonPanel />
+    </div>
+  );
+}
+
+Display.defaultProps = {
+  result: '0',
+};
```

Comment:
- Add this from `Display` component. This is the `App` component and doing this here will couple the two components. It's not a good practice